collect_app/src/main/java/org/odk/collect/android/activities/InstanceUploaderList.java

```diff
@@ -239,7 +254,7 @@ private void init() {
         listView.setChoiceMode(ListView.CHOICE_MODE_MULTIPLE);
         listView.setItemsCanFocus(false);
         listView.addOnLayoutChangeListener((v, left, top, right, bottom, oldLeft, oldTop, oldRight, oldBottom) -> {
-            uploadButton.setEnabled(areCheckedItems());
+            uploadButton.setEnabled(isReadyForUpload());
```

Comment:
I think we had decided to leave the button enabled and show the toast as needed to match existing behavior. If the button is disabled, the toast from `onUploadButtonsClicked` is never shown, right?

Reply:
What I thought based on the conversation we had in slack was that we would disable the button until the status for the `AutoSendWorker` has been set before enabling it. I am going to add that code to the `observer` that runs this `setEnabled` logic again if that behaviour makes sense. That's the reason for creating this method to toggle the button's state because it now depends on the `autoSendRunning` value.

Reply:
Oh, I was going too quickly and didn't realize `autoSendRunning` was a `Boolean` instead of a `boolean`. I guess I thought it was checking for `false`. Ok, this seems fine.